engine/combat/utility/sos_appraisal.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from enum import Enum
+
+from engine.combat.utility.core import Appraisal
+
+logger = logging.getLogger(__name__)
+
+
+# An action that a player can take. See AppraisalType
+class SoSAppraisalType(Enum):
+    NONE = 0
```

Comment:
Instead of using `= 0, = 1` etc. It's possible to use `= auto()`. auto is also imported `from enum`. It's better to use this unless the value is something that maps directly to something in memory.